scico/linop/xray/astra.py

```diff
@@ -65,13 +65,18 @@ def _project_coords(
     x_volume: np.ndarray, vol_geom: VolumeGeometry, proj_geom: ProjectionGeometry
 ) -> np.ndarray:
     """
-    Transform volume (logical) coordinates into world coordinates based
+    Project volume (logical) coordinates into detector coordinates based
```

Comment:
Is this a change of terminology from "world coordinates" to "detector coordinates"? If so, corresponding changes are required elsewhere for consistency.

Reply:
Not a change in terminology, the previous docstring was incorrect. 

Re volume vs logical: I hoped "logical" communicates that they are like integer indexes, but I'm removing it because I do not think it helps.

Reply:
It looks as if "logical" is still present.

Reply:
It would also be good to mention somewhere what "world" coordinates mean.

Reply:
Added to the module docstring after all.